html/SignUp.html

```diff
@@ -0,0 +1,70 @@
+<!DOCTYPE html>
+<html lang="ko">
+<head>
+  <meta charset="UTF-8">
+  <link rel="stylesheet" as="style" crossorigin href="https://cdn.jsdelivr.net/gh/orioncactus/pretendard@v1.3.9/dist/web/static/pretendard.min.css" />
+  <meta name="viewport" content="width=device-width, initial-scale=1.0">
+  <link rel="stylesheet" href="https://cdnjs.cloudflare.com/ajax/libs/normalize/8.0.1/normalize.min.css"/>
+  <title>SignUp</title>
```

Comment:
💊 제안
타이틀은 브라우저 탭에 표시되는 정보로 주로 어떤 사이트의 어떤 페이지에 머무르고 있는지를 표시합니다.
{사이트명} - {페이지명} 과 같은 형식으로 기입하셔서 유저에게 자세한 정보를 제공하시는 것을 추천드려요!

```suggestion
  <title>판다마켓 - 회원가입</title>
```